includes/site-health/audit-enqueued-assets/helper.php

```diff
@@ -22,7 +22,8 @@ function perflab_aea_enqueued_js_assets_test(): array {
 	 * If the test didn't run yet, deactivate.
 	 */
 	$enqueued_scripts = perflab_aea_get_total_enqueued_scripts();
-	if ( false === $enqueued_scripts ) {
+	$bytes_enqueued   = perflab_aea_get_total_size_bytes_enqueued_scripts();
+	if ( false === $enqueued_scripts || false === $bytes_enqueued ) {
```

Comment:
This could be a big fix, since it is possible that the transient for `perflab_aea_get_total_size_bytes_enqueued_scripts()` could be empty but not for `perflab_aea_get_total_enqueued_scripts()`.